app/services/plugins/entry/on_project_creation.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Plugins
+  module Entry
+    class OnProjectCreation
+      def initialize(project_info)
+        @project_info = project_info
+      end
+
+      def call
+        aws_user = AwsIntegration::User.new(@project_info.uniq_name)
+
+        access_key_hash = access_key_for_user(aws_user)
+        create_plugin_info!(access_key_hash)
+
+        aws_user.add_registry_access
+        aws_user.add_s3_access(repo_prefix: s3_repos_prefix)
+      end
```

Comment:
An Interactor would be perfect here. Create User and pass it to the interactor

What do you think?

Reply:
It seems like it. But I think it'll be rewritten later anyway, since plugins system will include more than one aws plugin, so we can discuss interactors during further refactoring